File: cpp/io/sol.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

int three_sum(int a, int b, int c){
    int res=a+b+c;
    return res;
}

int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int a,b,c;
    cin>>a>>b>>c;
    cout<<three_sum(a,b,c);
    return 0;
}

File: python3/compress-string/sol.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
from itertools import groupby
if __name__=="__main__":
	S=input()

	for i,j in groupby(map(int,list(S))):
		print("({}, {})".format(len(list(j)),i), end=" ")
	print()